src/test/java/com/integrivideo/pages/ChatPage.java

```diff
@@ -47,6 +47,8 @@ public void clickSendButton() {
         clickOn(sendButton);
     }
 
+    public void inviteUsers() { inviteButton.click(); }
```

Comment:
we already have public String getInviteLink()
Either delete that method or this one